version1/02 - Object types.md

```diff
@@ -400,7 +404,8 @@ The following notation is used in the tables below:
 |           | ref       | refPrimaerKlasse              |          |             |                          |  |      |  x   |   x   |      |  x   |   x    | <p>&bull; references a **Klasse** from the primary **Klassifikasjonssystem** of the **Arkivdel**</p>                                                                                        |
 |           | ref       | refSekundaerKlasse            |          |             |                          |  |      |      |   x   |      |  x   |   x    | <p>&bull; references a **Klasse** from any **Klassifikasjonssystem**</p>                                                                                                                    |
 |           | ref       | refEksternId                  |          |             |                          |  |      |      |   x   |      |  x   |        |                                                                                                                                                                                             |
-|           | ref       | refNoekkelord                 |          |             |                          |  |      |      |   x   |      |  x   |        | &nbsp;                                                                                                                                                                                      |
```

Comment:
Let's remove the `&nbsp;`s from the rest of the tables too (`AbstraktMappe` and `AbstraktRegistrering`).

Reply:
Removed.